libraries/abstractions/common_io/CMakeLists.txt

```diff
@@ -1,11 +1,11 @@
-afr_module()
+afr_module(INTERFACE)
```

Comment:
Are these changes needed for shadow demo?

Reply:
seems not necessary, I remember that's a replica from MQTT PR. will remove it.